public/achievementInfo.php

```diff
@@ -237,16 +303,30 @@ function updateAchievementTypeFlag(typeFlag) {
             echo "<div class='devbox mb-3'>";
             echo "<span onclick=\"$('#devboxcontent').toggle(); return false;\">Dev (Click to show):</span><br>";
             echo "<div id='devboxcontent' style='display: none'>";
+            RenderStatusWidget();
 
-            if ($permissions >= Permissions::Developer) {
-                RenderStatusWidget();
-                echo "<li>Set embedded video URL:</li>";
+            if ($permissions >= Permissions::Developer || ($isSoleAuthor && $permissions >= Permissions::JuniorDeveloper && $achFlags === AchievementType::Unofficial)) {
+                echo "<div>Update achievement details:</div>";
                 echo "<table><tbody>";
-                echo "<input type='hidden' name='a' value='$achievementID' />";
-                echo "<input type='hidden' name='f' value='2' />";
-                echo "<tr><td>Embed:</td><td style='width:100%'><input id='embedurlinput' type='text' name='v' value='$embedVidURL' style='width:100%;'/></td></tr>";
+                echo "<tr><td>Title:</td><td style='width:100%'><input id='titleinput' type='text' name='t' value='$achievementTitle' style='width:100%' maxlength='64';></td></tr>";
+                echo "<tr><td>Description:</td><td style='width:100%'><input id='descriptioninput' type='text' name='d' value='$desc' style='width:100%' maxlength='256';></td></tr>";
+                echo "<tr><td>Points:</td><td style='width:100%'><input id='pointsinput' type='number' name='p' value='$achPoints' style='width:100%' min='0' max='100';'></td></tr>";
```

Comment:
maybe use constants for 0 and 100?

Reply:
Do you feel it would be better to make new constants for min/max points or to access the first/last element in the VALID_POINTS const using something like this `VALID_POINTS[array_key_first(VALID_POINTS)];` and `VALID_POINTS[array_key_last(VALID_POINTS)];`?

Reply:
I believe php has functions for returning the min/max value from an array.